### ISSUE TITLE
Optimize the 'utility' helper

### ISSUE THREAD

**Issue by luchian** [pixeltrove/spark-css#1]
There seems to be room for improvement with respect to how the 'utility' helper is defined. Code that outputs utility classes is duplicated multiple times across the mixin whereas writing it once would lead to a clearer, less error-prone system.

**Comment by luchian** [comment]
All utility classes have the same basic structure.
```
.u-#{$alias}-#{$variant}#{$query-name-with-separator} {
  #{$property}: #{$value};
}
```
Consequently, this code fragment should be repeated only a small number of times throughout the 'utility' helper while the values each variable takes on are determined by conditional statements.

**Comment by luchian** [comment]
Using a single class declaration template as a source for all utilities doesn't seem possible. The best way to simplify the code is to employ multiple 'if' statements that check for keywords and generate utilities based on an appropriate class template.

**Comment by luchian** [comment]
Addressed in 38977f919264bfc32c28542b2d45449fb7cb0830.
